### ISSUE TITLE
Add MANIFEST.in With LICENSE (Conda Support)

### ISSUE THREAD

**Issue by pmlandwehr** [datafolklabs/cement#405]
Hey-lo,

I'm [building a version](https://github.com/conda-forge/staged-recipes/pull/1370#discussion_r76538266) of `cement` using [`conda`](http://conda.pydata.org) for [conda-forge](http://conda-forge.github.io). When possible, we try to include a link to the license file in the `meta.yaml` specification; doing so requires both:
- a license file be included in the package
- that license be indexed in an explicit [`MANIFEST.in`](https://docs.python.org/2/distutils/sourcedist.html#manifest-related-options) file.

Would you consider adding an explicit `MANIFEST.in` and making sure that `LICENSE`is included in it?

**Comment by derks** [comment]
I'd be happy to add a `MANIFEST.in` file ... would you please provide a suggested example of it (for Cement) so that I can ensure it includes what is needed (properly) for Conda?

Thanks.

**Comment by pmlandwehr** [comment]
Sure: this is untested, but should approximately work:
```
recursive-include *.py
include setup.cfg
include LICENSE README.md CONTRIBUTORS
include *.txt
```
Other packages with `MANIFEST` / `MANIFEST.in` files include [`requests`](https://github.com/kennethreitz/requests) and [`happybase`](https://github.com/wbolster/happybase)

**Comment by derks** [comment]
@pmlandwehr adding a `MANIFEST.in` which will be available in `stable/2.10.4` (which I'm working on releasing shortly).
